### DISCUSSION TITLE
Can't deploy even the simplest instance on 1.57.0

### DISCUSSION THREAD

**Discussion by mogoman** [jaegertracing/jaeger#5653] [Q&A] [answered]
I've been running the Jaeger operator from Helm chart 2.53.0 (app version 1.52.0) and all was fine, but after upgrading to 2.54.0 (app 1.57.0) I can't deploy jaeger instances anymore, the status of the instance always stays blank. Just to test, I downgraded via Helm to 2.53.0 and it immediately started working:

```
$ helm list -n observability | grep operator
jaeger-operator	observability	6       	2024-06-18 19:47:05.261400195 +0000 UTC	deployed	jaeger-operator-2.54.0	1.57.0
$ kubectl get jaeger -n observability
NAME       STATUS   VERSION   STRATEGY   STORAGE   AGE
simplest                                           5m3s

## downgraded Helm to 2.53.0

$ helm list -n observability | grep operator
jaeger-operator	observability	7       	2024-06-18 19:52:45.100310008 +0000 UTC	deployed	jaeger-operator-2.53.0	1.52.0
$ kubectl get jaeger -n observability
NAME       STATUS    VERSION   STRATEGY   STORAGE   AGE
simplest   Running   1.52.0    allinone   memory    5m30s
```

Helm is installed with all defaults. Any idea why this happens considering everything else remains the same?

**Comment by asimerel** [comment, marked as answer]
Because clusterrole doesn't have permissions for "ingressclasses" on 2.54.0
If you check the operator logs you can see the error about missing permission.

When you manually add this "ingressclasses" permission to clusterrole it will start working.

Related Issue: https://github.com/jaegertracing/helm-charts/issues/549

**Reply by mogoman** [reply]
thanks, I tested by updating the clusterrole per https://github.com/bouchardmathieu-qc/jaegertracing-helm-charts/commit/75bc3df52e04cce68cea84da5b81ebb33952e6f3 which fixes the problem temporarily for me. 

Edit:
Similar PR on different chart https://github.com/bouchardmathieu-qc/jaegertracing-helm-charts/pull/1

**Reply by yurishkuro** [reply]
>  bouchardmathieu-qc/jaegertracing-helm-charts

these are not official Jaeger charts
